models/pediatric_abdominal_ct_segmentation/configs/metadata.json

```diff
@@ -0,0 +1,101 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
+    "version": "0.4.3",
+    "changelog": {
+        "0.4.3": "add support for TensorRT conversion and inference",
```

Comment:
Hi @EmergentBehaviour , as a new bundle, I think we should only keep only one version in the changelog, could you modify here?

Reply:
changelog shortened